projects/plugins/jetpack/extensions/blocks/conversation/edit.js

```diff
@@ -29,9 +28,6 @@ const TRANSCRIPTION_TEMPLATE = [
 
 function ConversationEdit( { className, attributes, setAttributes } ) {
 	const { participants = [], showTimestamps, className: classNameAttr } = attributes;
-	const containerRef = useRef();
```

Comment:
let's remove the `useRef` import some lines above.

Reply:
Thanks! I removed it 👍